configs/sites/hercules/packages.yaml

```diff
@@ -144,7 +144,7 @@ packages:
   qt:
     externals:
     - spec: qt@5.15.8
-      prefix: /apps/spack-managed/gcc-11.3.1/qt-5.15.8-d47tsna6f5dylcpblkfgw4gpn2cucihd
+      prefix: /apps/spack-managed/gcc-12.2.0/qt-5.15.8-gayzhaahclvlybiiykwj2cym4vo33w6x
```

Comment:
Note to self, this needs to be reverted. But it has no significance, since `qt` isn't used in spack-stack directly.